python/commons/lab.py

```diff
@@ -0,0 +1,59 @@
+import sys
+import os
+import numpy as np
+
+import datajoint as dj
+
+schema = dj.schema('common_lab', locals())
+
+@schema
+class Paths(dj.Lookup):
```

Comment:
Why are you implementing this as a DataJoint table? It would make sense for you to implement this as a straight up object without inheriting from `dj.Lookup` - just use `class Paths` header if anything.